src/renderer/component/common/spinner.jsx

```diff
@@ -1,14 +1,14 @@
+// @flow
```

Comment:
Changed the spinner to not actually be a spinner, but might as well keep the name as is